### PR TITLE
Nicer app icon screen

### PR DESCRIPTION
**Pull request by mozzius**
https://github.com/user-attachments/assets/b3d770b9-d31f-48e5-b2c6-1ebc5a6ce0d2

### CODE REVIEW

File: src/screens/Settings/AppIconSettings.tsx [hunk 1]
```diff
@@ -1,252 +1,182 @@
...
-              <Text
-                style={[a.text_center, a.font_bold, a.text_md, a.mt_md]}
-                // for Classic™
-                emoji>
-                {icon.name}
-              </Text>
-            </View>
+        </Group>
+        <Text style={[a.text_xl, a.mt_lg, a.mb_md, a.font_heavy]}>
+          <Trans>Bluesky+</Trans>
```
Comment (estrattonbailey) [inline src/screens/Settings/AppIconSettings.tsx:76]: Do we need a translation for this one :P

File: src/screens/Settings/AppIconSettings.tsx [hunk 2]
```diff
@@ -1,252 +1,182 @@
...
-        name: _('Dark'),
-        iosImage: () => {
-          return require(`../../../assets/app-icons/ios_icon_default_dark.png`)
-        },
-        androidImage: () => {
-          return require(`../../../assets/app-icons/android_icon_default_dark.png`)
-        },
-      },
-    ]
+function Row({
```
Comment (estrattonbailey) [inline src/screens/Settings/AppIconSettings.tsx:137]: Nit: this and the Group feel kinda redundant to split out, could just inline, but nbd

File: src/screens/Settings/AppIconSettings.tsx [hunk 3]
```diff
@@ -1,252 +1,182 @@
...
+  const [currentAppIcon, setCurrentAppIcon] = useState(() =>
+    getAppIconName(DynamicAppIcon.getAppIcon()),
+  )
+
+  const onSetAppIcon = (icon: string) => {
+    if (isAndroid) {
+      const next =
+        sets.defaults.find(i => i.id === icon) ??
+        sets.core.find(i => i.id === icon)
+      Alert.alert(
```
[the remark this answers is not in the document]
Comment (mozzius) [reply src/screens/Settings/AppIconSettings.tsx:29]: We absolutely can, happy to switch that out

File: src/screens/Settings/AppIconSettings.tsx (hunk 2 above)
Comment (mozzius) [reply src/screens/Settings/AppIconSettings.tsx:137]: Yeah my feeling was this'll make it much easier to add additional sections

File: src/screens/Settings/AppIconSettings.tsx (hunk 1 above)
Comment (mozzius) [reply src/screens/Settings/AppIconSettings.tsx:76]: haha good question! have you just left it untranslated elsewhere?

**Comment by mozzius** [comment]
- that's our lightest shade (contrast_25). maybe we need a contrast_10
- yeah on iOS that's the system API giving that notification. Kinda silly

**Comment by pfrazee** [comment]
One change I made:

CleanShot 2024-12-05 at 19 09 15

**Comment by pfrazee** [comment]
|Before|After|
|-|-|
|CleanShot 2024-12-05 at 19 16 40|CleanShot 2024-12-05 at 19 17 06|

**Comment by mozzius** [comment]
50px was entirely arbitrary, no particular reason or significance

**Review by estrattonbailey — CHANGES_REQUESTED** [review]
Fixed web issues, but I think we've got a circular import in here somewhere on native now. Gonna pause for something else and come back 👍

**Review by estrattonbailey — APPROVED** [review]
Ope nvm, was just a case insensitive filepath on native

**Pull request merged**